src/services/connectionService.ts

```diff
@@ -129,18 +140,28 @@ const addDefaultConnections = async () => {
   }
 
   const firm24Name = 'Firm24'
-  const firm24 = parties.find((party: IConnectionParty) => party.name === firm24Name)
-  if (!firm24) {
+  const firm24CorrelationId = 'sphereon.com'
```

Comment:
It has the same correlationId as sphereon (website is firm24.com)

Reply:
fixed